src/lights/base/base_light.cpp

```
@@ -2,10 +2,10 @@
 #include "scene/scene.hpp"
 #include "renderer/window/render_context.hpp"
 
-mr::Light::Light(const Scene &scene, const Vec3f &color, size_t ubo_size)
+mr::Light::Light(Scene &scene, const Vec3f &color, size_t ubo_size)
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)